docs/pages/jpscore/jpscore_operativ.md

```diff
@@ -0,0 +1,187 @@
+---
+title: Operational models
+keywords: simulation
+tags: [jpscore, model]
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_operativ.html
+summary: Several operational models are implemented in jpscore. An operational model defines how pedestrians moves from one time step to the next.
+last_updated: Dec 21, 2019
+---
+
+## Introduction
+In the definition of agent's properties it is mandatory to precise the number of the model to be used e.g.:
+
+```xml
+ <agents operational_model_id="n">
+```
+The definition of any model parameter is composed of two different
+sections:
+
+- **model_parameters**: Model specific parameter. See below in the different model sections.
+- **agent_parameters**: These parameter are mainly specific for the shape of pedestrians
+    or other pedestrian properties like desired speed, reaction time etc.
+
+## Model parameters (in general)
+- `<solver>euler</solver>`
+     - The solver for the ODE. Only *Euler*. No other options.
+- `<stepsize>0.001</stepsize>`:
+     - The time step for the solver. This should be choosed with care. For force-based model it is recommended to take a value between $$ 10^{-2} $$ and $$10^{-3}$$ s.
+       For first-order models, a value of 0.05 s should be OK.
+       A larger time step leads to faster simulations, however it is too risky and can lead to
+numerical instabilities, collisions and overlapping among pedestrians.
+     - Unit: s
+- `<periodic>0</periodic>`
+     - Set to 1 if a system with closed boundary conditions should be simulated. Default setting is 0.
+     - This option is only implemented in *Tordeux2015* and is very geometry-specific (only for corridors) with predefined settings. 
+
+- `<exit_crossing_strategy>3</exit_crossing_strategy>`
+     - Positive values in $$[1, 9]$$. See [Direction strategies](jpscore_direction.html) for the definition of the strategies.
+
+- `<linkedcells enabled="true" cell_size="2"/>`
+     - Defines the size of the cells. This is important to get the neighbors of a pedestrians, which
+       are all pedestrians within the eight neighboring cells. Larger cells, lead so slower simulations, since
+       more pedestrian-pedestrian interactions need to be calculated.
+     - Unit: m
+
+
+## Agent's parameter (in general)
+The *agent parameters* are mostly identical for all models. Exceptions will be mentioned explicitly.
+
+The parameters that can be specified in this section are Gauss distributed (default value are given).
+
+### Desired speed
+- `<v0 mu="1.2" sigma="0.0" />`
+    - Desired speed
+    - Unit: m/s
+- `<v0_upstairs mu="0.6" sigma="0.167" />`
+    - Desired speed upstairs
+    - Unit: m/s
+- `<v0_downstairs mu="0.6" sigma="0.188" />`
+    - Desired speed downstairs
+    - Unit: m/s
+- `<v0_idle_escalator_upstairs mu="0.6" sigma="0.0" />`
+    - Speed of idle escalators upstairs
+    - Unit: m/s
+- `<v0_idle_escalator_downstairs mu="0.6" sigma="0.0" />`
+    - Speed of idle escalators downstairs
+    - Unit: m/s
```

Comment:
Maybe also include the table from the rimea tests with handbook values here?

Reply:
done.